resources/assets/less/bem/contest-user-entry.less

```diff
@@ -0,0 +1,63 @@
+.contest-user-entry {
+  height: 150px;
+  background: #313D40;
+  margin: 5px;
+  width: calc(50% ~'-' 20px);
```

Comment:
something about `padding: (@spacing * 2)` in entry-list and `calc(50% ~'-' (2*margin))` because otherwise it's a bit confusing.